fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -739,6 +744,12 @@ public boolean getUseScanBlockCache() {
     @VarAttr(name = QUERY_CACHE_ENTRY_MAX_ROWS)
     private long queryCacheEntryMaxRows = 409600;
 
+    @VarAttr(name = QUERY_CACHE_PRIMARY_KEYS_HOT_PARTITION_NUM)
+    private int queryCachePrimaryKeysHotPartitionNum = 3;
```

Comment:
Add some documents for these two variables?

Reply:
Comment has been already added at line 299~300